tests/e2e/OWNERS

```diff
@@ -1,10 +1,4 @@
-# See the OWNERS docs at https://go.k8s.io/owners
-
-
-reviewers:
-- sashrith
-- kavyashree-r
-
 approvers:
-- sashrith
-- kavyashree-r
+- openshift-storage-maintainers
+component: "Storage"
+subcomponent: Kubernetes External Components
```

Comment:
We were happy without this file for last six months (since `9a30e5998` removed it). Why do we need this file now?

Reply:
Because upstream added it, but our validation system does not like `OWNERs` file with unknown/non-redhat users.. There were some validations that were failing.